src/retractions.jl

```diff
@@ -143,13 +159,13 @@ Compute the inverse retraction, a cheaper, approximate version of the
 Result is saved to `X`.
 
 Inverse retraction method can be specified by the last argument, defaulting to
-[`LogarithmicInverseRetraction`](@ref). See the documentation of respective manifolds for
+`default_inverse_retraction_method(M)`. See the documentation of respective manifolds for
```

Comment:
```suggestion
[`default_inverse_retraction_method`](@ref)`(M)`. See the documentation of respective manifolds for
```
this way the reference to the function is also linked. Can you adapt that for all occurences?